content/en/network_monitoring/devices/snmp_traps.md

```diff
@@ -48,7 +53,12 @@ network_devices:
       privProtocol: "AES" # choices: DES, AES (128 bits), AES192, AES192C, AES256, AES256C
 ```
 
-**Note**: Multiple v3 users and passwords are supported as of Datadog Agent `7.51` or higher.
+  **Note**: Multiple v3 users and passwords are supported as of Datadog Agent `7.51` or higher.
```

Comment:
```suggestion
   **Note**: Multiple v3 users and passwords are supported as of Datadog Agent `7.51` or higher.
```

Reply:
Thanks let me try this! I was struggling with the markdown to get the formatting right as well :)